packages/caravan-health/src/privacy.test.ts

```diff
@@ -282,40 +315,65 @@ describe("Privacy metric scoring", () => {
 
   describe("Transaction Topology Score", () => {
     it("Calculates the transaction topology score based on the spend type", () => {
```

Comment:
might be worth changing this into a vector based test using test.each. then you can name them for each spend/tx type you're testing and consolidate the test code (also makes it easier to add more test cases in the future).